src/drivers/distance_sensor/ll40ls/ll40ls.cpp

```diff
@@ -99,6 +99,7 @@ extern "C" __EXPORT int ll40ls_main(int argc, char *argv[])
 	int ch;
 	using ThisDriver = LidarLiteI2C;
 	BusCLIArguments cli{true, false};
+	cli.orientation = distance_sensor_s::ROTATION_DOWNWARD_FACING;
```

Comment:
You could also directly set the default in https://github.com/PX4/Firmware/blob/master/platforms/common/include/px4_platform_common/i2c_spi_buses.h#L114.